### DISCUSSION TITLE
2 suggestions to help quiet down some repeated questions and get you some help

### DISCUSSION THREAD

**Discussion by DoubleStrike** [supermerill/SuperSlicer#4321] [General]
First, I want to say I appreciate the level of work that is going into the 2.7 merge and that it is a major work effort.

I would like to suggest at least every couple of weeks of updating the front page to say roughly how far along the process you feel you are and what your best-guess estimate is for the completion of the 2.7 merge work.  That would help more folks to see that progress is being made if they don't understand how to look at commit logs.  **It doesn't have to be super-exact, but even a very high-level estimate would be very useful every couple of weeks or so.**

Second, don't feel bad about asking for help with any grunt-work or tedious tasks.  By farming some of those out to the community, you could get some help and be able to focus on higher-value tasks in some cases.

Hopefully you take this in the spirit it is intended, which is to try to both keep you sane, and to help the community understand what the timeline looks like as it evolves.

**Comment by DoubleStrike** [comment]
Just to add, with increasing numbers of folks with multi-tool printers (especially the XL), the toolhead options introduced in 2.6 are critical for us, which basically completely locks us out of using 2.5, unfortunately.

**Reply by supermerill** [reply]
> I would like to suggest at least every couple of weeks of updating the front page to say roughly how far along the process you feel you are and what your best-guess estimate is for the completion of the 2.7 merge work. That would help more folks to see that progress is being made if they don't understand how to look at commit logs. It doesn't have to be super-exact, but even a very high-level estimate would be very useful every couple of weeks or so.

It's very bothersome to update the readme.
I can add a link to the dev channel of the discord.
I'll see if I can use the gihub "project" a bit more, so we can see the progress a bit better.

> Second, don't feel bad about asking for help with any grunt-work or tedious tasks. By farming some of those out to the community, you could get some help and be able to focus on higher-value tasks in some cases.

It's a bit problematic. I can't force someone to do something in a timeframe. And if I tell loud that something is avaialble for evryone to work on to help, and someone start to work on it, then I can't really rush it in the meantime and say "nevermind, I did it in the meantime, you began to work on it for nothing". I can but I feel bad about that.

> Hopefully you take this in the spirit it is intended, which is to try to both keep you sane, and to help the community understand what the timeline looks like as it evolves.

Thanks. I'm not good at managing people, and myself at all. This is complicated for me.
